plugins/slack_webhooks/app/views/samson_slack_webhooks/_fields.html.erb

```diff
@@ -2,7 +2,7 @@
   <legend>Slack</legend>
 
   <% stage = form.object %>
-  <% stage.slack_webhooks.build unless stage.slack_webhooks.last.try(:webhook_url).blank? %>
+  <% stage.slack_webhooks.build unless stage.slack_webhooks.size > 0 && stage.slack_webhooks.last.webhook_url.blank? %>
```

Comment:
this is meant to not add a blank hook when a update/create failed, so not super important ...